tests/js/client/load-balancing/load-balancing-key-generator-noauth-cluster.js

```diff
@@ -140,15 +172,39 @@ function KeyGeneratorSuite () {
           let result = sendRequest('POST', url, /*payload*/ {}, {}, i % 2 === 0);
           assertEqual(result.status, 202);
           let key = result.body._key;
-          assertTrue(key > lastKey || lastKey === null, { key, lastKey });
+          assertTrue(key > lastKey || lastKey === null, {key, lastKey});
           lastKey = key;
         }
       } finally {
         db._useDatabase("_system");
         db._dropDatabase(cn);
       }
     },
-    
+
+    testAutoincrementOnMultipleShards: function() {
+      try {
+        db._create(cn, {numberOfShards: 5, keyOptions: {type: "autoincrement"}});
+      } catch (error) {
+        assertEqual(error.code, 501);
```

Comment:
I would be in favor of using symbolic error names here, e.g.
```
assertEqual(error.errorNum, errors.ERROR_CLUSTER_UNSPUPPORTED.code);
```
(not tested, but something along these lines - additionally, `errors` needs to be defined first).